### ISSUE TITLE
404 Error - Fresh Install Nginx Proxy Apache Server Ubuntu 18.04

### ISSUE THREAD

**Issue by jehboi** [BookStackApp/BookStack#3412]
### Attempted Debugging

- [X] I have read the debugging page

### Searched GitHub Issues

- [X] I have searched GitHub for the issue.

### Describe the Scenario

After installing the BookStack on my development server (same stack as production), I attempted an install on production. Everything seemingly went smoothly - just like dev - but when I finally went to the site, it returns an Apache 404.

I have validated my Nginx configuration and Apache virtual host configuration and it all mirrors dev and I can find no discernible difference. I have validated permissions as well. I can access static resources (e.g. user_avatar.png).

I have searched similar issues but none have yielded a solution. Any assistance is appreciated!

### Exact BookStack Version

v22.04

### PHP Version

7.4

### Hosting Environment

Ubuntu 18.04 (Bionic), Nginx Reverse Proxy (forced HTTPS; LetsEncrypt), Apache 2

**Comment by ssddanbrown** [comment]
Hi @jehboi,

Based on the following comment:

>  I can access static resources (e.g. user_avatar.png).

It would seem to me that PHP might not be enabled on Apache, might be worth checking that mod-php is enabled and active at that layer.

The usage of both Apache and Nginx do complicate things here, if not the above it might be worth ensuring requests are properly hitting the apache layer by checking apache logs and/or stopping apache to confirm the error browser response changes.

**Comment by jehboi** [comment]
Thanks @ssddanbrown - I verified that PHP is indeed enabled and functioning. If I replace public/index.php with an echo 'Hello World' file it is served correctly.

Scanning configurations for both dev and prod has yielded nothing, so I have reconfigured Nginx to serve BookStack directly. This works. So it's something about the proxy setup (only in prod) that BookStack doesn't like. 

There is nothing in the logs. I will try some other things but at least I found a short term solution.
